package.json

```diff
@@ -86,7 +86,6 @@
     "chart.js": "^1.1.1",
     "cookie-session": "^2.0.0-alpha.1",
     "dataloader": "^1.2.0",
-    "dotenv": "^2.0.0",
```

Comment:
Moving from dependencies to devdependencies can lead to runtime errors if there is potential for usage directly in application code. Using [dotenv with the -r flag](https://www.npmjs.com/package/dotenv#preload) works well in some cases, but here I suggest just keeping dotenv in dependencies since we're using nodemon.

Reply:
I haven't looked carefull yet, but AFAICT, dotenv shouldn't ever be used in production. I'd like to figure out why that's happening before deciding to leave it in dependencies.

Reply:
Dotenv will never override already set environment variables ([docs](https://www.npmjs.com/package/dotenv#what-happens-to-environment-variables-that-were-already-set)) so in general I think it's safe but not preferable to use in production.

If we add code paths that could be hit in deployed applications then we should also make dotenv a production dependency IMO, though this is also avoidable if we use a wrapper script for `src/server/index.js` in dev or the `node --require` flag.

Reply:
Ooooh, I didn't realize it had that safeguard. Good to know that it at least won't wipe out already set vars. I still want to figure out where the actual problem is happening as it is (shouldn't be hard, I just haven't worked on it), but I'll keep that in mind.

Reply:
I suspect that `NODE_ENV` was set to `development` in @shakalee14's deployment, and because claudia only installs production dependencies it caused a runtime exception.

Reply:
Sounds highly plausible!